rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/InterpretedLanguageContributions.java

```diff
@@ -237,8 +243,24 @@ private static boolean hasContribution(ISet contributions, String name) {
 
     private static CompletableFuture<Boolean> getContributionParameter(
             CompletableFuture<ISet> contributions, String name, String parameter) {
+        return getContributionParameter(contributions, name, parameter, VF.bool(false)).thenApply(IBool::getValue);
+    }
+
+    @SuppressWarnings("unchecked")
+    private static <T extends IValue> CompletableFuture<T> getContributionParameter(CompletableFuture<ISet> contributions, String name, String parameter, T defaultVal) {
+        return contributions.thenApply(c -> {
+            var contrib = getContribution(c, name);
+            if (contrib == null) {
+                return defaultVal;
+            }
 
-        return contributions.thenApply(c -> isTrue(getContribution(c, name), parameter));
+            var val = contrib.asWithKeywordParameters().getParameter(parameter);
+            try {
+                return (T) val;
+            } catch (ClassCastException e) {
+                return defaultVal;
+            }
```

Comment:
Are we sure that due to type-erasure it doesn't fail if you wanted an `IList` but got an `IString` since `T` at runtime will be `Object`? So this will only fail with a `null` value, but that might not even throw a `ClassCastException`?